contract/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "agoric-basics-contract",
+  "version": "0.1.0",
+  "private": true,
+  "description": "Agoric Basics Contract",
+  "type": "module",
+  "scripts": {
+    "test": "ava --verbose",
+    "lint": "eslint '**/*.js'",
+    "lint:fix": "eslint --fix '**/*.js'"
+  },
+  "devDependencies": {
+    "@agoric/deploy-script-support": "^0.10.4-u12.0",
+    "@agoric/eslint-config": "dev",
+    "@endo/bundle-source": "^2.8.0",
+    "@endo/eslint-plugin": "^0.5.2",
+    "@endo/init": "^0.5.60",
+    "@endo/promise-kit": "0.2.56",
+    "@endo/ses-ava": "^0.2.44",
+    "@jessie.js/eslint-plugin": "^0.4.0",
+    "@typescript-eslint/eslint-plugin": "^6.7.0",
+    "@typescript-eslint/parser": "^6.7.0",
+    "agoric": "^0.21.2-u12.0",
+    "ava": "^5.3.0",
+    "eslint": "^8.47.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-jessie": "^0.0.6",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-ava": "^14.0.0",
+    "eslint-plugin-github": "^4.10.0",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-jsdoc": "^46.4.3",
+    "eslint-plugin-prettier": "^5.0.0",
+    "import-meta-resolve": "^2.2.1",
+    "prettier": "^3.0.3",
+    "prettier-plugin-jsdoc": "^1.0.0",
+    "type-coverage": "^2.26.3",
+    "typescript": "~5.2.2"
+  },
+  "dependencies": {
+    "@agoric/ertp": "^0.16.3-u12.0",
+    "@agoric/zoe": "^0.26.3-u12.0",
+    "@endo/far": "^0.2.22",
+    "@endo/marshal": "^0.8.9",
+    "@endo/patterns": "^0.2.5"
+  },
+  "ava": {
+    "files": [
+      "test/**/test-*.js"
+    ],
+    "timeout": "10m"
+  },
+  "keywords": [],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Agoric/dapp-agoric-basics"
+  },
+  "author": "Agoric",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/Agoric/dapp-agoric-basics/issues"
+  },
+  "homepage": "https://github.com/Agoric/dapp-agoric-basics#readme",
+  "eslintConfig": {
+    "parserOptions": {
+      "sourceType": "module",
+      "ecmaVersion": 2021
+    },
+    "extends": [
+      "@agoric"
+    ]
+  },
```

Comment:
This looks good. As a heads up, `dapp-offer-up` has a backlog item to expand this configuration to include type checking, and `no-void` + `no-floating-promises` rules: https://github.com/Agoric/dapp-offer-up/issues/35#issuecomment-1918030472

This has been blocked by lack of type support in the current releases, which prevents us from checking this sort of thing into CI: https://github.com/Agoric/dapp-offer-up/issues/31 (This assumption may need to be revisited, as the two eslint rules seem separate from typechecking)

Reply:
Got it, created https://github.com/Agoric/dapp-agoric-basics/issues/2 to track this